vue-model-api/src/internal/ReactiveINodeJS.ts

```diff
@@ -84,6 +84,10 @@ export class ReactiveINodeJS implements INodeJS {
       : unreacitveNode;
   }
 
+  remove(): void {
+    this.unreactiveNode.remove();
```

Comment:
Question: Do we have to invoke `track` here?

Reply:
No, because the parent already `tracked` when you accessed the child. The parrent will `trigger` when the child is removed.
I see that you already verifyed it in `ReactiveINodeJS.test.ts`.

Reply:
Thanks for the explanation :)